### ISSUE TITLE
[Question] is it possible to determine if app uses face id vs touch id?

### ISSUE THREAD

**Issue by stantoncbradley** [naoufal/react-native-touch-id#66]
Is there a way to determine where the user's phone supports touch id vs face id? For example, we would like to determine whether our UI prompts user for facial recognition or touch id.  thanks for the great lib!

**Comment by edcrampin** [comment]
TouchID.isSupported should return a promise that contains a string of either "FaceID" or "TouchID", which should allow you to determine this. However, latest version of this code on NPM does not match up with the latest Git version so if you're having issues with this, that might be why.

As per latest Git version, you should be able to use like this:
```
TouchID.isSupported()
  .then(biometryType => {
    // Success code
    if (biometryType === 'FaceID') {
        console.log('FaceID is being used.');
    } else {
        console.log('TouchID is being used.');
    }
  })
  .catch(error => {
    // Failure code
    console.log(error);
  });
```

**Comment by naoufal** [comment]
@stantoncbradley Just released 4.0.0 which lets you test for FaceID like @edcrampin described.
